media-sound/supercollider/supercollider-3.13.0.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="https://github.com/supercollider/supercollider/releases/download/Versio
 LICENSE="GPL-2 gpl3? ( GPL-3 )"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="cpu_flags_x86_sse cpu_flags_x86_sse2 debug emacs +fftw gedit +gpl3 jack qt5 server +sndfile static-libs vim webengine X zeroconf"
+IUSE="cpu_flags_x86_sse cpu_flags_x86_sse2 debug emacs +fftw gedit +gpl3 jack qt5 server +sndfile static-libs vim webengine X +zeroconf"
```

Comment:
We should probably install the vim files unconditionally per https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301. It's unclear to me if Emacs is actually used at build time. If it isn't used to byte-compile anything, we should install those unconditionally too. Ditto gedit files.

But I'm not going to hold up this PR on that because it's totally broken right now in gentoo.git, and we want this working, so thank you!

Reply:
Will do! Will start on that tomorrow for Vim as well as gedit (checked the CMakeLists and it doesn't compile anything and tested to confirm as well).

Looking at the [CMakeLists](https://github.com/supercollider/scel/blob/d8296a9af54d145703726328e22b9e89a08476af/el/CMakeLists.txt) file under scel/el, it looks like it compiles .el files and will throw an error if emacs isn't installed. Am I safe in assuming that probably means we shouldn't make it unconditional?

Speaking of emacs, I noticed the [readme](https://github.com/supercollider/scel/blob/d8296a9af54d145703726328e22b9e89a08476af/README.md) for it says this: 

> Whatever option you choose, make sure not to mix installation methods. In particular, do not install the Quark if you already have the supercollider-emacs package or if you compiled SuperCollider with the -DSC_EL=ON option. Otherwise you will get an error from SuperCollider about duplicated classes.

I'll try to put a check in for that in my next commit as well! That actually might fix [an open bug](https://bugs.gentoo.org/733496).

Thank you both for being so helpful!